src/site/layouts/support_resources_article_listing.drupal.liquid

```diff
@@ -21,7 +21,7 @@
                 {% assign snippet = article.fieldIntroTextLimitedHtml %}
 
                 {% if article.entityBundle == 'q_a' %}
-                  {% assign snippet = article.fieldAnswer.0.entity.fieldWysiwyg %}
+                  {% assign snippet = article.fieldAnswer.entity.fieldWysiwyg %}
```

Comment:
This is the fix for the article snippets not showing for QA pages.

Reply:
I have a very random liquid template question:

Let's say I have an array called items (`[1, 2, 3]`) and another array called otherItems (`[4, 5, 6]`) and I want to do something like this:

```liquid
{% for item in items %}
  {% assign itemIndex = forloop.index %}

  {% for otherItem in otherItems %}
    {{ item[itemIndex] }}
  {% endfor %}
{% endfor %}
```

I'm just trying to figure out if you can do `{{ array[forloop.index] }}` in liquid, @youngfreezyVA and I weren't able to get that to work